spec/models/show_time_spec.rb

```diff
@@ -1,5 +1,75 @@
 require "rails_helper"
 
 RSpec.describe ShowTime, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let!(:movie){
+    FactoryBot.create :movie
+  }
```

Comment:
Có nhất thiết phải sử dụng let! không em, phân biệt let và let!

Reply:
let gọi mỗi khi chạy testcase
let! - tự động gọi kể cả khi không chạy testcase 
cả giá trị của let và let! đều được cache lại trong it.